From c1ed3733986486f3255e77dec47ae1d4b2836e2f Mon Sep 17 00:00:00 2001
From: Joachim Wiberg <troglobit@gmail.com>
Date: Mon, 11 Mar 2024 15:46:38 +0100
Subject: [PATCH 3/4] Fix #397: drop ttinit() for PID 1
Organization: Addiva Elektronik

After reports from the field, see issue #397, of lockups at reboot,
we've decided to drop this code from PID 1.  It was added before the 4.x
series, when the current progress output was introduced.  For the older
style progress it served a purpose since the placement of [OK]/[FAIL]
was on the right hand side.

Signed-off-by: Joachim Wiberg <troglobit@gmail.com>
---
 src/log.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/log.c b/src/log.c
index ea49705..b776525 100644
--- a/src/log.c
+++ b/src/log.c
@@ -42,8 +42,6 @@ static int loglevel = LOG_INFO;
 
 void log_init(void)
 {
-	ttinit();
-
 	if (debug)
 		loglevel = LOG_DEBUG;
 	else
@@ -53,13 +51,6 @@ void log_init(void)
 /* If we enabled terse mode at boot, restore to previous setting at shutdown */
 void log_exit(void)
 {
-	/*
-	 * Unless in debug mode at shutdown, Reinitialize screen,
-	 * terminal may have been resized at runtime
-	 */
-	if (!debug)
-		ttinit();
-
 	enable_progress(1);
 }
 
-- 
2.34.1

